docpages/example_programs/the_basics/embeds.md

```diff
@@ -1,6 +1,6 @@
 \page embed-message Sending Embeds
 
-If you've been in a server and used a bot or even seen a message from a bot, you might have seen a special messages, often sent by these bots. These are called embeds! In this section, we will show how to create an embed and reply to a user, using our newly created embed!
+If you've been in a server and used a bot or even seen a message from a bot, you might have seen a special message type, often sent by these bots. These are called embeds! In this section, we will show how to create an embed and reply to an user, using our newly created embed!
```

Comment:
`a user` here is correct because the phonetics of the word, the change is incorrect